src/index.scss

```diff
@@ -1,2 +1 @@
-@use 'the-new-css-reset/css/reset.css';
-@use './styles/_main.scss';
+@use './styles/main.scss';
```

Comment:
@mikewuu Should we follow this convention? https://sass-lang.com/guide

> Partials
> You can create partial Sass files that contain little snippets of CSS that you can include in other Sass files. This is a great way to modularize your CSS and help keep things easier to maintain. A partial is a Sass file named with a leading underscore. You might name it something like _partial.scss. The underscore lets Sass know that the file is only a partial file and that it should not be generated into a CSS file. Sass partials are used with the @use rule.

Reply:
Yep, I did try this yesterday, but ran into a couple roadblocks. Will spin off into a new issue to refactor to use `@use`

Reply:
https://github.com/seamapi/react/issues/84